dev-games/ogre/ogre-1.10.12.ebuild

```diff
@@ -2,6 +2,8 @@
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=6
+CMAKE_REMOVE_MODULES="yes"
```

Comment:
please keep the emptyline after EAPI, instead you can drop the one above inherit

Reply:
fixed